image-search/Models/ActivityIndicator.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  ActivityIndicator.swift
+//  image-search
+//
+//  Created by Renato Vieira on 22/10/19.
+//  Copyright © 2019 Stant. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+class ActivytyIndicator: UIViewController {
```

Comment:
This is not a model, so it shouldn't be on "Models" group. 
My suggestion to you is that you create a group called "Utils" and put it there.

Reply:
Done